webapp/src/components/EstateDetailPage/EstateDetail.css

```diff
@@ -134,4 +134,23 @@
     margin-top: 0px;
     margin-bottom: 20px;
   }
+
+  .ui.stackable.grid {
+    width: auto;
+    margin-left: 0!important;
+    margin-right: 0!important;
+  }
+  
+  .ui.stackable.grid:not(.vertically)>.row {
+      margin: 0;
+      padding: 0;
+  }
+
+  .ui.grid>.stackable.stackable.row>.column, .ui.stackable.grid>.column.grid>.column, .ui.stackable.grid>.column.row>.column, .ui.stackable.grid>.column:not(.row), .ui.stackable.grid>.row>.column, .ui.stackable.grid>.row>.wide.column, .ui.stackable.grid>.wide.column {
```

Comment:
Refer to https://github.com/decentraland/marketplace/pull/586/files/d2a083875d94f9f98566921f6e4f4e8dc0e5b8da#r226081500